iam_profile_faker/factory.py

```diff
@@ -229,37 +226,37 @@ def get_public_email_address():
                                    else (self.fake.pyint(), self.fake.pyint()))
 
         values = {
-            'LastName': self.fake.last_name(),
-            'Preferred_Name': self.fake.name(),
-            'PreferredFirstName': self.fake.first_name(),
-            'LegalFirstName': self.fake.first_name(),
-            'EmployeeID': employee_id,
-            'businessTitle': self.fake.job(),
-            'IsManager': self.fake.pybool(),
-            'isDirectorOrAbove': self.fake.pybool(),
-            'Management_Level': get_management_level(),
-            'HireDate': self.fake.date(pattern="%Y-%m-%d", end_datetime=None),
-            'CurrentlyActive': random.choice(['0', '1']),
-            'Entity': self.fake.company(),
-            'Team': '{} team'.format(self.fake.color_name()),
-            'Cost_Center': '{} - {}'.format(self.fake.pyint(), self.fake.job()),
-            'WorkerType': random.choice(['Employee', 'Seasonal', 'Geocontractor']),
-            'LocationDescription': random.choice([
+            'last_name': self.fake.last_name(),
```

Comment:
the camelcase is not because of coding style reasons, it matches the export from HRIS thus the profile v2 schema.

Reply:
This is still compatible with the v2 schema since it's just an object of `AccessInformationValues`. The two examples of the v2 schema have the hris data looking like this. I don't have strong feelings about this commit overall. Until we know how this looks exactly (maybe those fields are normalized somewhere before reaching the v2 schema) I would prefer to keep it this way. That said, I can also drop the commit

Reply:
This is still compatible with the v2 schema since it's just an object of `AccessInformationValues`. The two examples of the v2 schema have the hris data looking like this. I don't have strong feelings about this commit overall. Until we know how this looks exactly (maybe those fields are normalized somewhere before reaching the v2 schema) I would prefer to keep it this way. That said, I can also drop the commit

Reply:
I would leave this as is and also map it 1 to 1 to graphql.